feature-libs/commerce-quotes/assets/translations/en/commerce-quotes.i18n.ts

```diff
@@ -31,4 +29,16 @@ export const commerceQuotesStates = {
 export const commerceQuotes = {
   commerceQuotesList,
   commerceQuotesStates,
+  commerceQuotesCommons: {
```

Comment:
I think we could go with "commons" as we're already prefixing with context of `commerceQuotes`.